tests/wpt/web-platform-tests/http/basic-auth-cache-test.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<meta charset="utf-8">
+<link rel="match" href="basic-auth-cache-test-ref.html">
+<html>
+  <div id="auth"> </div>
+  <div id="noauth"> </div>
+  <script type="text/javascript">
+    var authImg = '<img src="http://testuser:testpass@' + window.location.host + '/http/resources/securedimage.py">';
+    document.getElementById('auth').innerHTML = authImg;
+    setTimeout(function() {
```

Comment:
Rather than a timeout (which could cause intermittent failures if the first image finishes loading before the timeout fires), we should:
* add `class="reftest-wait"` to the `<html>` element
* add an onload handler to the first image, which starts the second image load
* add an onload handler to the second load which removes the reftest-wait class